js/components/wysiwyg.js

```diff
@@ -81,13 +81,19 @@ Fliplet.FormBuilder.field('wysiwyg', {
       tinymce.DOM.setStyle(this.labelElement, 'display', '');
     },
     onPlaceholderFocus: function () {
+      var outlineContainer = document.querySelector('.mce-container-body.mce-stack-layout');
+      outlineContainer.classList.add('focus-outline-active');
```

Comment:
```suggestion

      outlineContainer.classList.add('focus-outline-active');
```

Also, check that the `outlineContainer` is not falsey.

Reply:
Fixed